binder/postBuild

```diff
@@ -16,18 +16,24 @@
 #
 set -euo pipefail
 
+export VERSION=1.22.4
```

Comment:
Could this be renamed to `YARN_VERSION`?  I didn't know what _version_ it referenced until seeing the commands below.